### PR TITLE
update DG - jinglin

### LINKED ISSUE #55: Update DG - Jing Lin

### PR DESCRIPTION
**Pull request by aqcd**
resolves #55

### CODE REVIEW

File: docs/diagrams/sequenceDiagrams/AddQuantityToItemCommandSequenceDiagram.puml [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+activate InventoryinatorParser
+
+create AddQuantityToItemCommandParser
+InventoryinatorParser -> AddQuantityToItemCommandParser
+activate AddQuantityToItemCommandParser
+
+AddQuantityToItemCommandParser --> InventoryinatorParser
+deactivate AddQuantityToItemCommandParser
+
+InventoryinatorParser -> AddQuantityToItemCommandParser : parse("addq -n banana -q 50")
```
Comment (blackonyyx) [inline docs/diagrams/sequenceDiagrams/AddQuantityToItemCommandSequenceDiagram.puml]: Command Word is removed at this step, the input that is recieved by parser is `-n banana -q 50`

File: docs/diagrams/sequenceDiagrams/AddQuantityToItemCommandSequenceDiagram.puml [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+deactivate CommandResult
+
+EditItemCommand --> AddQuantityToItemCommand
+deactivate EditItemCommand
+
+AddQuantityToItemCommand --> LogicManager : result
+deactivate AddQuantityToItemCommand
+
+destroy CommandResult
+[<--LogicManager
```
Comment (blackonyyx) [inline docs/diagrams/sequenceDiagrams/AddQuantityToItemCommandSequenceDiagram.puml:78]: Command Result is Returned and parsed to the UI i think So its not destroyed

Screenshot 2020-10-19 at 9 00 17 AM

File: docs/diagrams/sequenceDiagrams/AddQuantityToItemCommandSequenceDiagram.puml [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+
+create CommandResult
+EditItemCommand -> CommandResult
+activate CommandResult
+
+CommandResult --> EditItemCommand
+deactivate CommandResult
+
+EditItemCommand --> AddQuantityToItemCommand
+deactivate EditItemCommand
```
Comment (blackonyyx) [inline docs/diagrams/sequenceDiagrams/AddQuantityToItemCommandSequenceDiagram.puml:72]: EditItem Command should also probably be destroyed after CommandResult is returned.

**Review by blackonyyx — CHANGES_REQUESTED** [review]
Some minor inaccuracies in Sequence Diagram need to be addressed

Overall format looks good.

Am not sure whether we need to express Exception Handling steps in Sequence flow diagrams. 

Can consider looking at this if you would like to do so. (Optional)
https://johanvergeer.github.io/posts/uml-sequence-diagram-exception-handling

**Comment by aqcd** [comment]
Opened #73 regarding above comment

File: docs/diagrams/sequenceDiagrams/AddQuantityToItemCommandSequenceDiagram.puml [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+
+AddQuantityToItemCommand --> LogicManager : result
+deactivate AddQuantityToItemCommand
+
```
Comment (blackonyyx) [inline docs/diagrams/sequenceDiagrams/AddQuantityToItemCommandSequenceDiagram.puml:76]: AddQuantityToItemCommand should also similarly be destroyed.

**Review by blackonyyx — APPROVED** [review]
LGTM after adding the one additional destroy mentioned.

**Pull request merged**